zio-http/src/main/scala/zio/http/Body.scala

```diff
@@ -207,14 +264,17 @@ object Body {
 
     override def isComplete: Boolean = false
 
-    override def write(ctx: Ctx)(implicit trace: Trace): Task[Boolean] =
-      ZIO
-        .attempt(unsafeAsync { (ctx, msg, isLast) =>
-          ctx.writeAndFlush(msg)
-          if (isLast) ctx.writeAndFlush(LastHttpContent.EMPTY_LAST_CONTENT): Unit
-          else ctx.read(): Unit
-        })
-        .as(true)
+    override private[zio] def asChunkUnsafe(implicit unsafe: Unsafe): Chunk[Byte] = {
+      implicit val trace = Trace.empty
+      val rtm            = Runtime.default
```

Comment:
Ditto here. Should just let Netty driver do this since the code is identical in both places.